cli/compose/types/types.go

```diff
@@ -125,7 +127,8 @@ type ServiceConfig struct {
 	Ulimits         map[string]*UlimitsConfig
 	User            string
 	Volumes         []ServiceVolumeConfig
-	WorkingDir      string `mapstructure:"working_dir"`
+	WorkingDir      string              `mapstructure:"working_dir"`
+	Isolation       container.Isolation `mapstructure:"isolation"`
```

Comment:
I think this should be `string`. Everywhere else we accept plain types and do the convert to swarm/api types in convert. This helps keep the compose config packages decoupled from the api.

I believe this change will remove the need for the custom transform in loader.go as well.

Reply:
I guess this was here previously because of validation. I should have looked closer when I was commenting the first time. Sorry about that.